pages/order.tsx

```diff
@@ -540,6 +536,9 @@ const Landing: NextPage = () => {
     setIsFocused(false);
   };
 
+  const SUGGESTED_RESPONSES = ['Place Order', 'Cancel Order', 
```

Comment:
Should be outside of the component since it's a constant, no need to reinitialize every render